scripts/generate-report.py

```
@@ -327,46 +765,118 @@ def add_styled_table(doc, headers, rows, col_widths=None):
-def build_executive_summary(doc, df, epoch, stats, date_label):
-    """Page 1: Executive summary with key metrics and governance state."""
-    # Title
-    title = doc.add_paragraph()
-    title.alignment = WD_ALIGN_PARAGRAPH.CENTER
-    run = title.add_run(f"Feed Quality Analysis \u2014 {date_label}")
+def build_title_page(doc, date_label, epoch):
+    """Title page with report name and subtitle."""
+    epoch_id = epoch.get("id", "?")
+
+    title = doc.add_paragraph(style="Title")
+    run = title.add_run(f"Feed Quality Analysis")
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)